aip/general/0123.md

```diff
@@ -71,6 +71,9 @@ message Topic {
 - Patterns **must** correspond to the [resource name][aip-122].
 - Pattern variables (the segments within braces) **must** use `snake_case`, and
   **must not** use an `_id` suffix.
+- Pattern variables **must** be the singular form of the resource type e.g. the
+  pattern variable representing a `Topic` resource ID is named `{topic}`.
+- Pattern variables **must** conform to the format `[a-z][_a-z0-9]*[a-z]`.
```

Comment:
Is there any reason that it cannot end with a number?

Reply:
Not that I know of, so I will add it to the last segment